pippo-less4j/src/main/java/ro/pippo/less4j/LessResourceHandler.java

```diff
@@ -39,37 +36,30 @@
 
     private static final Logger log = LoggerFactory.getLogger(LessResourceHandler.class);
 
-    private Map<String, String> sourceMap = new ConcurrentHashMap<>(); // cache
-
     private boolean compress;
+    private Map<String, String> sourceMap = new ConcurrentHashMap<>(); // cache
 
     public LessResourceHandler(String urlPath, String resourceBasePath) {
         super(urlPath, resourceBasePath);
-        this.compress = Application.get().getPippoSettings().isProd();
     }
 
     @Override
-    public URL getResourceUrl(String resourcePath) {
-        URL url = super.getResourceUrl(resourcePath);
-        if (url == null) {
-            return null;
-        }
-
+    protected void sendResource(URL resourceUrl, RouteContext routeContext) throws IOException {
         // clear cache for DEV mode
-        if (Application.get().getRuntimeMode().equals(RuntimeMode.DEV)) {
+        if (routeContext.getApplication().getPippoSettings().isDev()) {
             sourceMap.clear();
```

Comment:
Is it better to touch the cache (`sourceMap`) only when I am  in `PROD` mode (no `sourceMap.get()`, `sourceMap.put()` in this mode)?